src/features/multivariate_layer/helpers/createBivariateColorsForMVA.test.ts

```
@@ -0,0 +1,130 @@
+    id: 'A3',
+    color: 'rgba(228, 26, 28, 0.5)',
+  },
+  {
+    id: 'C1',
+    color: 'rgba(166, 239, 179, 0.5)',
+  },
+  {
+    id: 'C3',
+    color: 'rgba(102, 154, 112, 0.5)',
+  },
+];
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)